src/main/webapp/app/shared/vertical-progress-bar/vertical-progress-bar.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="vertical-progress-bar">
```

Comment:
why don't we use ngxcharts here?

Reply:
Huge performance overhead for a simple progress bar